tests/test_metrics.py

```diff
@@ -29,36 +29,71 @@ class Test_precision_recall:
     """ Ensure that precision and recall calculations are correct. """
 
     detections_a = np.array([[0, 0, 1, 1, 1]])
-    detections_b = np.array([[0, 0, 1, 1, 2]])  # identical box, but different class from a
-    detections_c = np.array([[1, 1, 2, 2, 1]])  # no overlap with a/b, different class from b
+    detections_b = np.array(
+        [[0, 0, 1, 1, 2]]
+    )  # identical box, but different class from a
+    detections_c = np.array(
+        [[1, 1, 2, 2, 1]]
+    )  # no overlap with a/b, different class from b
     empty_detect = np.empty((0, 5))
 
     @pytest.mark.parametrize(
-        ("prediction_detections", "truth_detections", "desired_precision", "description"),
-        [(detections_a, detections_a, 1.0, "identical boxes"),
-         (detections_a, detections_b, 0.0, "identical box with different class"),
-         (detections_a, detections_c, 0.0, "non-overlapping boxes"),
-         (detections_b, detections_c, 0.0, "non-overlapping boxes with different classes"),
-         (detections_a, empty_detect, 0.0, "empty truth and non-empty predictions"),
-         (empty_detect, detections_a, 1.0, "empty predictions and non-empty truth"),
-         (empty_detect, empty_detect, 1.0, "empty predictions and empty truth")])
-    def test_known_precision(self, prediction_detections, truth_detections, desired_precision, description):
+        (
+            "prediction_detections",
+            "truth_detections",
+            "desired_precision",
+            "description",
+        ),
+        [
+            (detections_a, detections_a, 1.0, "identical boxes"),
+            (detections_a, detections_b, 0.0, "identical box with different class"),
+            (detections_a, detections_c, 0.0, "non-overlapping boxes"),
+            (
+                detections_b,
+                detections_c,
+                0.0,
+                "non-overlapping boxes with different classes",
+            ),
```

Comment:
this is awful....it completely breaks the flow -- are there alternatives/can we just have black respect a slightly longer line length (like 100?)